product-scenarios/2-Bridging-systems-that-communicate-in-different-protocols/02-SynapseConfigProject/02-synapseConfig/pom.xml

```diff
@@ -1,13 +1,13 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <project xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd" xmlns="http://maven.apache.org/POM/4.0.0"
-         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance">
+    xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance">
   <modelVersion>4.0.0</modelVersion>
-  <groupId>org.wso2.ei.product-scenarios.scenario_01</groupId>
-  <artifactId>01-synapseConfig</artifactId>
+  <groupId>com.example.02-synapseConfig</groupId>
```

Comment:
Shall we add correct groupId? It should be `org.wso2.ei.product-scenarios.scenario_02`